server/server.py

```diff
@@ -166,8 +173,10 @@ def wrong_argument(message):
 def get_version():
   # Verify that tensorboard is running
   req_res = tb_request("logdir")
-  if not isinstance(req_res, str):
-    return req_res
+  try:
```

Comment:
Got it, should be fixed in https://github.com/torrvision/crayon/pull/11/commits/445882edab2829ea65f1ee420e9d3114eadf8faa.